src/resultset/resultset_record.c

```diff
@@ -29,7 +29,8 @@ Record* Record_FromGroup(const ResultSetHeader *resultset_header, const Group *g
             AR_ExpNode *agg_exp;
             Vector_Get(g->aggregationFunctions, agg_idx, &agg_exp);
             AR_EXP_Reduce(agg_exp);
-            r->values[i] = AR_EXP_Evaluate(agg_exp);
+            printf("AR EXP below should be constant, no need for record.\n");
```

Comment:
I don't know if I follow this - what should we be doing here?

Reply:
Forgot this printf, there's nothing we should do here, to compute agg_exp, we don't need a record as an aggregated expression is just a numeric value, so no lookups for node / edge is required.